src/ports/postgres/modules/convex/mlp_igd.py_in

```diff
@@ -1781,7 +1799,7 @@ class MLPMinibatchPreProcessor:
             summary_table_columns = summary_table_columns[0]
 
         required_columns = (self.DEPENDENT_VARNAME, self.INDEPENDENT_VARNAME,
-                            self.CLASS_VALUES, self.GROUPING_COL)
+                            self.CLASS_VALUES, self.GROUPING_COL, self.DEPENDENT_VARTYPE)
         if set(required_columns) <= set(summary_table_columns):
             self.preprocessed_summary_dict = summary_table_columns
         else:
```

Comment:
Can you please update the optimizer params in online docs?

Reply:
+1 will do